packages/app/src/update/helpers/version-formatter.helper.test.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { versionFormatter } from "App/update/helpers/version-formatter.helper"
+
+describe("Production release", () => {
+  test("returns provided version", () => {
+    expect(versionFormatter("1.2.3")).toEqual("1.2.3")
+  })
+})
+
+describe("Release candidate release", () => {
+  test("returns provided version", () => {
+    expect(versionFormatter("1.2.3-rc.1")).toEqual("1.2.3-rc.1")
+  })
+})
+
+describe("Daily release", () => {
+  test("returns formatted version", () => {
+    expect(versionFormatter("1.2.3-daily.2022.08.12")).toEqual(
+      "1.2.3-daily.2022.8.12"
```

Comment:
This formatter logic handles the issue on the OS CI side, and daily will be supported after marge? 🤔 
https://appnroll.atlassian.net/browse/CP-1003

Reply:
Yes

Reply:
So... What do you think to take this issue to the active sprint and pass to tests with this issue? 🤔